src/index.rst

```diff
@@ -92,3 +92,10 @@ Frequently Asked Questions
 * I found a vulnerability in Toltec or one of its packages, where can I report it?
 
   To inform the Toltec maintainers about a security issue, please follow the `security instructions <https://github.com/toltec-dev/organization/blob/main/docs/security.md>`_.
+  
+* My reMarkable seems to have become unresponsive, how do I fix it?
+
+  If you can't SSH in through USB, there are tools that can help you recovery your remarkable if it becomes unresponsive:
```

Comment:
I suggest adding a note that the USB interface corresponds to IP 10.11.99.1. From my experience it seems that not all users know this. Also, did you mean _recover_ instead of _recovery_?
```suggestion
  If you can't SSH in through USB (using IP ``10.11.99.1``), there are tools that can help you recover your remarkable if it becomes unresponsive:
```

Reply:
Thanks!

>  Also, did you mean _recover_ instead of _recovery_?

Err, yes. Whoops!